config/chewy.yml

```diff
@@ -5,3 +5,5 @@ test:
   prefix: 'test'
 development:
   host: 'localhost:9200'
+production:
+  host: 'https://f49b.es.eu-west-1.aws.found.io'
```

Comment:
I think it would make more sense to pass it in as an environment variable if we can. So here:
```
host: <%= ENV["ELASTICSEARCH_HOST_URL"] %>
```
And then add that to the Heroku instance as a config var in Settings.

It's usual practice to pass rails config fils through erb so this sort of dynamic processing can be done, but I can't find any documentation on the Chewy site that this will be done - so this might not work.

Reply:
Actually [it does look like the config file is passed to erb](https://github.com/toptal/chewy/blob/d8c7bf741805c4c0b658f0d1e542609b94145227/lib/chewy/config.rb#L134). So my suggestion should work.

Reply:
will give that a go, and have added the variable in Heroku